cg/nuget/cgmanifest.json

```diff
@@ -2670,7 +2670,7 @@
         "Type": "NuGet",
         "NuGet": {
           "Name": "runtime.osx-x64.BuildXL",
-          "Version": "1.99.99"
+          "Version": "2.0.1"

```

Comment:
@rijulluman-msft - this is unrelated to my change. Is it just due to staging of when we enabled the enforcement side of things?

Reply:
I haven't been in the loop about this - what does this indicate, cause if its not updated?

Reply:
It's automatically generated in dev builds. Soon, PR and rolling builds will fail if the cgmanifest file is out of sync with the packages that are fetched. The purpose of the file is to pump package information into the component governance system.

Reply:
yes, this is exactly because the cgmanifest file checked in with Rijul's change was a bit stale

Reply:
Thanks!

Reply:
Yes, looks like the file I generated was a bit stale